packages/front-end/components/Auth/InAppHelp.tsx

```diff
@@ -8,36 +8,33 @@ import { GBPremiumBadge } from "../Icons";
 import UpgradeModal from "../Settings/UpgradeModal";
 
 export default function InAppHelp() {
-  const config = useFeature("papercups-config").value;
+  const { app_id, script_content } = useFeature("pylon-config").value;
```

Comment:
The feature value will be `null` if there's a network problem or the feature is toggled off for some reason.  That will throw an exception here since you are using object destructuring on it.

Reply:
Ah, thank you!